source/Outfit.cpp

```diff
@@ -712,6 +714,42 @@ const Sprite *Outfit::FlotsamSprite() const
 
 
 
+const std::vector<std::pair<const Outfit *, double>> Outfit::GetResuppliedAmmo() const
+{
+	return resuppliedAmmo;
+}
+
+
+
+bool Outfit::AmmoResupplied(const Outfit *ammo) const
+{
+	bool isResupplied = false;
+	for(const std::pair<const Outfit *, double> &refilled : GetResuppliedAmmo())
+	{
+		const Outfit *refilledAmmo = refilled.first;
+		if(ammo == refilledAmmo)
+			isResupplied = true;
+	}
+	return isResupplied;
+}
+
+
+
+const double Outfit::GetResupplyCostMultiplier(const Outfit *ammo) const
+{
+	std::optional<double> minimumMultiplier = 1000.;
+	for(const std::pair<const Outfit *, double> &resupplied : GetResuppliedAmmo())
+	{
+		const Outfit *resuppliedAmmo = resupplied.first;
+		double costMultiplier = resupplied.second;
+		if(ammo == resuppliedAmmo && (costMultiplier < minimumMultiplier || minimumMultiplier == 0.))
```

Comment:
```suggestion
	optional<double> minimumMultiplier;
	for(const pair<const Outfit *, double> &resupplied : resuppliedAmmo)
	{
		const Outfit *resuppliedAmmo = resupplied.first;
		double costMultiplier = resupplied.second;
		if(ammo == resuppliedAmmo && (costMultiplier < minimumMultiplier || !minimumMultiplier))
```
(please correct me if I'm wrong on how std::optional works)